src/lib/index.tsx

```diff
@@ -62,6 +68,7 @@ const CopyMailTo = ({
   anchorStyles = {},
 }: {
   email: string;
+  theme?: string;
```

Comment:
Use [TypeScript String Literals](https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types) here for defining the types of theme.

Reply:
Resolved! Thanks for the suggestion @prateek3255